tests/testthat/output/iris--70.txt

```diff
@@ -1,9 +1,8 @@
-Source:     table<iris-output-test> [?? x 5]
-Database:   sqlite x.y.z [:memory:]
-Grouped by: Species
-Ordered by: Sepal.Length
-
-# S3: tbl_dbi
+# A tibble  : ?? x 5
+# Source    : table<iris-output-test> [?? x 5]
```

Comment:
Now aligning colons by adding space before them. Let me know if you prefer adding space after the colon, like in the original output.